Makefile

```diff
@@ -79,9 +88,22 @@ release : docker
 	@docker login -e $(DOCKER_EMAIL) -u $(DOCKER_USERNAME) -p $(DOCKER_PASSWORD)
 	@for build in $(builds) ; do \
 	  set -e; \
-	  tag=skycirrus/feed-$$build:$(git_rev) ; \
-	  latest_tag=skycirrus/feed-$$build:latest ; \
+	  tag=${repository}/feed-$$build:${build_version} ; \
+	  latest_tag=${repository}/feed-$$build:latest ; \
 	  docker tag $$tag $$latest_tag ; \
 	  docker push $$tag ; \
 	  docker push $$latest_tag ; \
 	done
+
+releasetotest : docker
+	@echo "== release docker image to the test repository"
+	@for build in $(builds) ; do \
+	  set -e; \
+	  image_tag=${repository}/feed-$$build:${build_version} ; \
+	  test_tag=${repository}/test/feed-$$build:${build_version} ; \
+	  latest_tag=${repository}/test/feed-$$build:latest ; \
+	  docker tag $$image_tag $$test_tag ; \
+	  docker tag $$test_tag $$latest_tag ; \
+	  docker push $$test_tag ; \
+	  docker push $$latest_tag ; \
+	done
```

Comment:
I'm not convinced we need or want this. Why can't you just manually do a docker push if you want to test it?

We could also add a test repo for building the branches and do that instead of allowing overrides of the variables.